.github/actions/multi-approvers/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "multi-approvers",
+  "version": "1.0.0",
```

Comment:
I think we should start at 0.1.0 probably, until we're confident to cut a 1.0. Per semver, we can make breaking changes for < 1.0.

Reply:
Done.